web/chat/chat-thread-background.react.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import * as React from 'react';
+
+import { threadInBackgroundChatList } from 'lib/shared/thread-utils';
+
+import ChatThreadList from './chat-thread-list.react';
+import css from './chat-tabs.css';
+
+export default function ChatThreadBackground() {
+  return (
+    <ChatThreadList
+      filterThreads={threadInBackgroundChatList}
+      emptyItem={EmptyItem}
+    />
+  );
+}
+
+function EmptyItem() {
+  return (
+    <div className={css.emptyItem}>
+      Background threads are just like normal threads, except they appear in
```

Comment:
Instead of copy-pasting this text, let's put it in `lib/shared/thread-utils.js`, and import it from both native and web. That way if we can change it in one place we'll change it in the other